client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/CRUDDocumentationIT.java

```diff
@@ -100,6 +100,7 @@
  */
 public class CRUDDocumentationIT extends ESRestHighLevelClientTestCase {
 
+    @SuppressWarnings("unused")
```

Comment:
Where is the actual unused variable here?

Reply:
L164-L167 in the "tag::index-response" block. All the variables mentioned there should be in the documentation snippet but are unused. Same in most other documentation tests.

Reply:
Could a dummy line be added to the snippet that "uses" the variables?

Reply:
That would be a lot of dummy lines, already five in this method, more in other test. I'm very much in favour of supressing them on the method in these particular cases.